scripts/js/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "sygma-substrate-pallet-scripts",
+  "version": "1.0.0",
+  "main": "index.js",
+  "dependencies": {
+    "@phala/typedefs": "^0.2.32",
```

Comment:
Do we have dependencies from phala?

Reply:
lol no we dont, it was a copy&paste, I will remove it